recipes/polyglotdb/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "polyglotdb" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89d2b4a19595a7bc14d1fba124afc63454baa9678275cd093d091d0430fc36c2
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

Comment:
This needs to be removed until https://github.com/MontrealCorpusTools/PolyglotDB/pull/167 is available.
```suggestion
```